main.tf

```diff
@@ -0,0 +1,136 @@
+locals {
+  app_name_and_env = "${var.app_name}-${var.app_env}"
+  name_tag_suffix  = "${var.app_name}-${var.customer}-${var.app_environment}"
+}
+
+/*
+ * Create cloudwatch log group for app logs
+ */
+resource "aws_cloudwatch_log_group" "cw_b2_fsbackup" {
+  name              = local.app_name_and_env
+  retention_in_days = 14
```

Comment:
Devon suggested increasing this to at least 45 days. Might be longer...I can't remember exactly.